buyer-api/test/facades/dataResponseFacade/closeDataResponse.test.js

```diff
@@ -1,9 +1,80 @@
 import test from 'ava';
-// import { closeDataResponse } from '../../../src/facades';
-
-test.skip('responds with error when orderAddress is invalid', t => t.fail());
-test.skip('responds with error when sellerAddress is invalid', t => t.fail());
-test.skip('responds true if dataResponse is already closed', t => t.fail());
-test.skip('if notariesToDemandAuditsFrom has something, it demands an audit', t => t.fail());
-test.skip('throws if audit is in progress', t => t.fail());
-test.skip('returns true if dataResponse is successfully closed', t => t.fail());
+import sinon from 'sinon';
+import client from 'request-promise-native';
+import { closeDataResponse } from '../../../src/facades';
+import * as notariesFacade from '../../../src/facades/notariesFacade';
+import * as utils from '../../../src/utils'; // { dataOrderAt }
+import config from '../../../config';
+
+const fakeAddress = '0x1234567890123456789123456789012345678901';
+
+test.afterEach.always(() => { sinon.restore(); });
+
+test.serial('responds with error when orderAddress is invalid', async (assert) => {
+  const error = await assert.throws(closeDataResponse('I will fail!', fakeAddress, undefined));
+  assert.is(error.message, 'Invalid order|seller address');
+});
+
+test.serial('responds with error when sellerAddress is invalid', async (assert) => {
+  const error = await assert.throws(closeDataResponse(fakeAddress, 'I will fail!', undefined));
+  assert.is(error.message, 'Invalid order|seller address');
+});
+
+test.serial('responds true if dataResponse is already closed', async (assert) => {
+  sinon.stub(utils, 'dataOrderAt').value(() => ({ methods: { getSellerInfo: () => ({ call: () => ['useless', 'useless', 'useless', 'useless', 'useless', '0x'] }) } }));
+  const callback = sinon.spy();
+  const result = await closeDataResponse(fakeAddress, fakeAddress, callback);
+  assert.true(result);
+  assert.false(callback.called);
+});
+
+test.serial('if notariesToDemandAuditsFrom has something, it demands an audit', async (assert) => {
```

Comment:
Would change this to: `demands audits from notaries in notariesToDemandAuditsFrom`